docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/OwnerUndefinedValidator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\va_gov_backend\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Validates the OwnerUndefined constraint.
+ */
+class OwnerUndefinedValidator extends ConstraintValidator {
```

Comment:
```suggestion
class FieldAdministationUndefinedValidator extends ConstraintValidator {
```
File needs to be renamed accordingly.